test/sslapitest.c

```diff
@@ -54,72 +54,63 @@ struct sslapitest_log_counts {
     unsigned int server_application_secret_count;
 };
 
-static void client_keylog_callback(const SSL *ssl, const char *line) {
+static void client_keylog_callback(const SSL *ssl, const char *line)
+{
     int line_length = strlen(line);
 
     /* If the log doesn't fit, error out. */
-    if ((client_log_buffer_index + line_length) > LOG_BUFFER_SIZE) {
-        printf("No room in client log\n");
+    if (client_log_buffer_index + line_length > sizeof(client_log_buffer)) {
+        TEST_info("Log too full");
         error_writing_log = 1;
         return;
     }
 
     strcat(client_log_buffer, line);
     client_log_buffer_index += line_length;
-    client_log_buffer[client_log_buffer_index] = '\n';
-    client_log_buffer_index += 1;
-
-    return;
+    client_log_buffer[client_log_buffer_index++] = '\n';
 }
 
-static void server_keylog_callback(const SSL *ssl, const char *line) {
+static void server_keylog_callback(const SSL *ssl, const char *line)
+{
     int line_length = strlen(line);
 
     /* If the log doesn't fit, error out. */
-    if ((server_log_buffer_index + line_length) > LOG_BUFFER_SIZE) {
-        printf("No room in server log\n");
+    if (server_log_buffer_index + line_length > sizeof(server_log_buffer)) {
+        TEST_info("Log too full");
```

Comment:
Same is in `client_keylog_callback`, compare with `sizeof(server_log_buffer) - 1`, and mention that it's the server log.